tmt/steps/provision/local.py

```diff
@@ -137,7 +151,8 @@ def execute(
         )
 
     def install_scripts(self, scripts: Sequence[tmt.steps.scripts.Script]) -> None:
-        self.debug("No installation of tmt scripts is needed on localhost.")
+        """Install scripts required by tmt"""
+        # no-op function
 
```

Comment:
Let's keep the previous debug message. It's still relevant, we do no install scripts on the local guest.